lib/sync_checker/formats/publication_check.rb

```diff
@@ -2,7 +2,12 @@ module SyncChecker
   module Formats
     class PublicationCheck < EditionBase
       def root_path
-        "/government/publications/"
+        case edition_expected_in_live.publication_type
```

Comment:
That would also support non-publication editions that are going through this and don't have `#publication_type`.